.github/workflows/integ-tests.yml

```diff
@@ -6,7 +6,37 @@ on:
       - develop
 
 jobs:
-  integ-tests:
+  go-tests:
+    runs-on: ubuntu-latest
+    environment:
+      name: prod
```

Comment:
not related to your change in particular, but could you change the environment name to use `integ-tests`? (The environment is already created in the settings). It seems weird that the environment is `prod`, when this is really only tests.  This for all the jobs.